packages/autobahn/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "autobahn",
+  "version": "19.7.3",
+  "description": "An implementation of The Web Application Messaging Protocol (WAMP).",
+  "main": "index.js",
+  "files": [
+    "index.js",
+    "/lib"
+  ],
+  "engines": {
+    "node": ">= 4.2.6"
+  },
+  "dependencies": {
+    "cbor": ">= 3.0.0",
+    "crypto-js": ">=3.1.8",
+    "msgpack5": ">= 3.6.0",
+    "tweetnacl": ">= 0.14.3",
+    "tweetnacl-sealedbox-js": ">=1.1.0",
+    "when": ">= 3.7.7",
+    "ws": ">= 1.1.4",
+    "sha3": ">=2.0.1",
```

Comment:
I think this dependency should be moved to `autobahn-xbr` as well. It was added together with the xbr feature in #433.

Reply:
Moved `sha3`, `uuid` and `tweetnacl-sealedbox-js` to autobahn-xbr package.